CHANGELOG for v1.x.x.md

```diff
@@ -2,6 +2,382 @@
+
+* #2795 [fixed] - Cart error merging if you authenticate having items with low stock
+
+* #2800 [fixed] - Add the ability to change Attribute Family for Products
+
+* #2804 [fixed] - There should be order review section instead of complete section on checkout page
+
+* #2805 [fixed] - the sku should be all in caps
+
+* #2819 [fixed] - Template overrides not working properly since upgrading to 1.1.0
+
+* #2822 [fixed] - set login on mobile number for costumer
+
+* #2844 [fixed] - showing product image of each color in configurable product
+
+* #2863 [fixed] - Search Product Name and Description
+
+* #2864 [fixed] - Role should we have view permission?
+
+* #2868 [fixed] - Cart (customer) address not persisted during the checkout
+
+* #2870 [fixed] - Should public folder be in gitignore
+
+* #2874 [fixed] - Order, payment process and payment metadata
+
+* #2885 [fixed] - getting the value of a custom attribute in shop side
+
+* #2888 [fixed] - Always Default locale should be selected when add new product for each channels
+
+* #2889 [fixed] - timezone drop down field is not visible in web installer
+
+* #2891 [fixed] - Problem exporting to xls and csv
+
+* #2893 [fixed] - When creating a shipment, display items invoiced
+
+* #2913 [fixed] - PauUmoney payment gateway integration
+
+* #2916 [fixed] - Minute tweak for breaking of theme assets.
+
+* #2922 [fixed] - Create Refund call Payment Method
+
+* #2926 [fixed] - bagisto_asset function not able to find files in custom package
+
+* #2929 [fixed] - How can set global value for items per page in datagrid
+
+* #2931 [fixed] - Customer pays order in PayPal but there is no record in bagisto
+
+* #2936 [fixed] - change the admin route for another
+
+* #2941 [fixed] - how to remove the magnifier from product .
+
+* #2942 [fixed] - Randomize New and Featured Products
+
+* #2948 [fixed] - How to remove Velocity Logo from admin
+
+* #2949 [fixed] - failed to migrate with new database using installer
+
+* #2950 [fixed] - multiple error message on installer
+
+* #2951 [fixed] - Getting error Base table channel not found
+
+* #2964 [fixed] - Exception when buying non stockable item via API
+
+* #2969 [fixed] - Cancel icon is not visible in velocity theme for customer order detail
+
+* #2972 [fixed] - can add to homescreen on mobile device in velocity theme
+
+* #2974 [fixed] - Thumbnails are not generating on mobile ifproduct has more than 4 photos
+
+* #2976 [fixed] - Security warning: HTTPS on images with SSL installed.
+
+* #2979 [fixed] - can we have functionality to insert fake data
+
+* #2980 [fixed] - After uploading on server rupees symbol not showing≈
+
+* #2982 [fixed] - Column not found: 1054 Unknown column 'default_name' in 'field list' (v1.1.2)
+
+* #2985 [fixed] - Product category is not saving
+
+* #2986 [fixed] - How to enable Vue debug
+
+* #2987 [fixed] - mult address
+
+* #2988 [fixed] - Facebook Customer Chat for Laravel
```

Comment:
Why are general questions being added to this list as "fixed" as it doesn't contain any change in Bagisto?

Reply:
Yes, you are right, I am removing general questions from change log. Thanks for pointing out